app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -7,9 +7,16 @@ class CurationConcerns::GenericWorksController < ApplicationController
   include Sufia::WorksControllerBehavior
 
   before_action :check_recent_uploads, only: [:show]
+  after_action  :notify_rdr, only: [:create]
 
   set_curation_concern_type GenericWork
 
+  def notify_rdr
+    @msg = "https://deepblue.lib.umich.edu"+ main_app.curation_concerns_generic_work_path(curation_concern) ;
+    email = WorkMailer.deposit_work(@current_user.email,@msg)
```

Comment:
We don't want to be sending email on behalf of the user.  I can see a utility to this, but it's too close to email spoofing.